test_data/parser/paths/4.expect

```diff
@@ -0,0 +1,12 @@
+error: unexpected token at 1..7
+NODE_ROOT 0..7 {
+  NODE_IDENT 0..1 {
+    TOKEN_IDENT("a") 0..1
+  }
+  NODE_ERROR 1..7 {
```

Comment:
It would be nice if the error were more isolated (rather than consuming the rest of the expression in code like `a${b}/c + /tmp`), but I think we can figure that out once we merge some of the very helpful error handling code that @oberblastmeister has been working on

Reply:
I'll need to think of this later today, but actually we should split 0.10 in 0.10 and 0.11. The reason for this is that I'd like to have a rnix-parser which doesn't provide invalid syntax-errors for new language-features that are part of the current stable Nix (2.4) which will be stable Nix in the upcoming 21.11.

That said, yes, I'd love to have @oberblastmeister's changes at some point, definitely, but I think we should cut 0.10 a bit earlier than planned IMHO :)